dashboard/db/schema.rb

```diff
@@ -2205,17 +2207,6 @@
     t.index ["word", "definition"], name: "index_vocabularies_on_word_and_definition", type: :fulltext
   end
 
-  add_foreign_key "ap_school_codes", "schools"
```

Comment:
Add back the foreign keys?

Reply:
Oh these too! Not sure how these are being generated